project-document/docs/domain-model/group-context/container-aggregate/food.md

```diff
@@ -1,14 +1,33 @@
 # Food
 
+## Food ID (Identifier)
+
+The food ID is the UUID.
+
 ## Food (Entity)
 
+The food is food in a container. The food cannot exist without the name and the category, but the unit, quantity, and expiry are optional.
+
+## Food Name
+
+The length of the food name must be less than equal 30 and greater than 0.
 
 ### Unit (Value Object)
 
 The length of unit must be less than equal to 20 characters.
+The unit can be changed after setting.
 
 ### Quantity (Value Object)
 
-The quantity must be more than equal 0.
-The quantity can be the float. In that case, the effective digit is 2 and the less than that is discarded.
-The reason for adopting discarding instead of rounding is to avoid increasing quantity which is an unnatural phenomenon.
\ No newline at end of file
+The quantity must be more than 0.
```

Comment:
In my understanding, the quantity can be null.

Reply:
That's true. I should change it to more than 0, or null.